config/testgrids/kubernetes/sig-testing/config.yaml

```diff
@@ -131,6 +136,10 @@ dashboards:
         url: https://github.com/kubernetes/test-infra/compare/<start-custom-0>...<end-custom-0>
 
 - name: sig-testing-kind
+  dashboard_tab:
```

Comment:
we care much more about this being in the conformance-all and conformance-kind dashboards